lumibot/tools/polygon_helper.py

```diff
@@ -92,8 +89,8 @@ def get_price_data_from_polygon(
     # print(f"\nGetting pricing data for {asset} / {quote_asset} with '{timespan}' timespan from Polygon...")
 
     # RESTClient connection for Polygon Stock-Equity API; traded_asset is standard
-    # Add "trace=True" to see the API calls printed to the console for debugging
-    polygon_client = RESTClient(api_key)
+    # Add "trace=True" to see the API capolygon_clientlls printed to the console for debugging
```

Comment:
typo capolygon_clientlls -> calls

Reply:
Thanks! I already fixed it: https://github.com/Lumiwealth/lumibot/pull/455/commits/a8faabc64f3c922672d6ddfc84a97daf9e18edcf